website/common/locales/en/loginIncentives.json

```diff
@@ -6,13 +6,10 @@
   "totalCount": "<%= count %> total count",
```

Comment:
Is this string used anywhere? I can't find it

Reply:
I think I confused it with occurrences in functions:
e.g.,

 in: `website/client/src/components/userMenu/profileStats.vue`
```javascript
...
  <strong>{{ $t('petsFound') }}:</strong>
                {{ totalCount(user.items.pets) }}
              </div>
...
```
in: `website/client/src/components/userMenu/stats.vue`
```javascript
...
  {{ $t('mountsTamed') }}:
              {{ totalCount(user.items.mounts) }}
...
```

Reply:
I just removed this string from all languages files